primary/src/primary.rs

```diff
@@ -189,7 +183,7 @@ impl Primary {
             certificate_store.clone(),
             synchronizer,
             signature_service.clone(),
-            consensus_round.clone(),
+            rx_new_consensus_round.clone(),
```

Comment:
you can also do `tx_newconsensus_round.subscribe()` to create new Receiver, in this case don't think there is much of a difference but just to raise awareness in case you haven't seen it.

Reply:
Yeah, that is nice, and used it in the main code (not tests).